src/metpy/calc/tools.py

```diff
@@ -887,6 +888,16 @@ def azimuth_range_to_lat_lon(azimuths, ranges, center_lon, center_lat, geod=None
     else:
         g = geod
 
+    try:  # convert range units to meters
+        ranges = ranges.to('meters').m
```

Comment:
Been trying to use this more:
```suggestion
        ranges = ranges.m_as('meters')
```

Reply:
Got it.